app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -21,16 +21,12 @@ def title
   end
 
   def description
```

Comment:
Did you try removing this completely? I think it was only necessary because we were essentially saying "instead of using the description, use a modified version of the description that has the HTML breaks stripped."